### PR TITLE
Add jailed validator log

### PR DESCRIPTION
**Pull request by viquezclaudio**
## Pull request checklist

- [ ] All tests pass. The project builds and runs.
- [ ] I have resolved any merge conflicts.
- [ ] I have resolved all `clippy` and `rustfmt` warnings.

### CODE REVIEW

File: blockchain/src/history/validity_store.rs [hunk 1]
```diff
@@ -119,7 +119,7 @@ impl ValidityStore {
         db_txn.remove(&self.block_txns, &block_number);
     }
 
-    /// Prunes the validity store keeping only 'validity_window_blocks'
+    /// Prunes the validity store keeping only 'validity_window_blocks' + 'blocks_per_batch' blocks
```
Comment (hrxi) [inline blockchain/src/history/validity_store.rs]: Doesn't really belong into this commit, but looks good to me nevertheless.

**Review by hrxi — APPROVED** [review]
LGTM, thanks.

**Comment by hrxi** [comment]
Rebased to sign the commits. I also split them since I had the opportunity.

**Review by Eligioo — APPROVED** [review]
LGTM, rebasing again to get it merged

**Pull request merged**